### ISSUE TITLE
Use git tags for releases, update CHANGELOG

### ISSUE THREAD

**Issue by StephenBrown2** [pawamoy/mkdocstrings#29]
I'm enjoying the work so far, but it's a bit hard to follow the progress, as there are no releases on GitHub, and no tags in lieu of "official" releases.

Could you perhaps start tagging releases with the version number and pushing them to GitHub? Oh, and also update the `CHANGELOG.md`? I realize it's difficult sometimes when you're working stream-of-consciousness as I often do.

**Comment by pawamoy** [comment]
Exactly, I didn't want to spend time writing a changelog for such a quickly moving project.
I was planning to start properly using git tags and updating the changelog once I was satisfied with the architecture. Which should happen with the next release 😄

**Comment by StephenBrown2** [comment]
Tags at minimum is great, let's me see the changes with something like https://github.com/pawamoy/mkdocstrings/compare/v0.6.0...v0.7.0 much easier.

**Comment by pawamoy** [comment]
Done! I was able to recover the right commits for each version using some git/grep-fu and the releases dates.
